Testscripts/Windows/NTTTCP-MONITOR-THROUGHPUT.ps1

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+param(
+	[String] $TestParams,
+	[object] $AllVMData,
+	[object] $CurrentTestData
+)
+
+function Main {
+	param (
+		$TestParams
+	)
+	$currentTestResult = Create-TestResultObject
+	$scriptFile = "./ntttcp_monitor_throughput.sh"
+	$log_dir = "/root/ntttcp_monitor"
+
+	$serverVMData = $allVMData | Select-Object -First 1
+	$server_internal_ip = $serverVMData.InternalIP
+	$serverResult = Run-LinuxCmd -ip $serverVMData.PublicIP -port $serverVMData.SSHPort -username $global:user `
+		-password $password -command "chmod +x *.sh && $scriptFile --log_dir '${log_dir}' --server_internal_ip '${server_internal_ip}'" -runAsSudo -ignoreLinuxExitCode
```

Comment:
I see your intention of "chmod +x *.sh". But you remember this introduced another side-effect - adding execution permission on constants.sh and other *.sh file without your intention. @johnsongeorge-w said using "bash script_file" better.

Reply:
no impact to this test case. not sure others.

Reply:
It is true to have no impact of the functionality. However, this shared open source project has the recommended way of coding style for all participants. This helps the source code in a uniform way helping other people read the code easily. When we work on LISAv2, we committed to following the LISAv2 policy and rules to help each other for the same work goal.
@harz566 There were thousands of comments about coding style in the past PR. I don't see we have any exceptions.